puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/test/clone_foreman.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -ex
+
+rm -rf foreman/
+git clone https://github.com/theforeman/foreman --branch "${foreman_branch}"
```

Comment:
Thanks for pointing that out - I have pushed a fix to add that parameter to the job with a default.